src/options/mkoptions.py

```diff
@@ -1428,7 +1439,7 @@ def mkoptions_main():
                     m[0].alternate = False
                     alias.alternate_for = m[0]
                     del g_long_cache[alias.long]
-            check_long(alias.filename, alias.lineno, alias.long)
+            check_long(alias.filename, 0, alias.long)
```

Comment:
I think the two calls to `check_long` changed in this PR are the only ones. Should we maybe remove the lineno argument from the function?

Reply:
Ditto